internal/genconfig/generate_config_test.go

```diff
@@ -423,6 +423,53 @@ resource "tfcoremock_simple_resource" "empty" {
   list   = null
   map    = null
   single = null
+}`,
+		},
+		"jsonencode_wrapping": {
+			schema: &configschema.Block{
+				Attributes: map[string]*configschema.Attribute{
+					"juststr": {
+						Type:     cty.String,
+						Optional: true,
+					},
+					"jsonobj": {
+						Type:     cty.String,
+						Optional: true,
+					},
+					"jsonarr": {
+						Type:     cty.String,
+						Optional: true,
+					},
+				},
```

Comment:
Can we add a test case with a sensitive object, and one with a sensitive object field?

Reply:
:+1: on the test case.  My only remaining question is if the generated config should include sensitive values?  What was the behavior before?

Reply:
Generated config doesn't include sensitive values, it just sets null with a comment: https://github.com/opentofu/opentofu/blob/main/internal/genconfig/generate_config.go#L161

Regarding the test case - I was adding a couple of additional tests when I discovered a panic (in existing code). It is caused by non-sensitive objects with an inner field value marked as sensitive. 

I'm investigating if that's a real case and how to fix it if so.